modDesc.xml

```diff
@@ -32,7 +32,7 @@ Différents modes d'utilisation ont été ajoutés depuis les premières version
 			<![CDATA[Этот мод может быть использован для создания сети маршрутов для транспортных средств для автономного вождения. После настройки вы можете указать трактору, стоящему где-угодно рядом с сетью, проехать в любую точку, например, в магазин, поле №1 или в точку продажи.]]>
 		</ru>
 	</description>
-	<version>2.0.0.5</version>
+	<version>2.0.0.6.16</version>
```

Comment:
Same here, also 2.0.0.6.16 is not allowed.
It would be a 2.0.0.7 if you go higher then .6 but not add another Number.

Reply:
I used the extra number for my local version tracking during testing. I figured that the AD team would discard my modDesc.xml edits and make their own for version control at the root level.